autonomy/cli/helpers/deployment.py

```diff
@@ -115,6 +142,31 @@ def run_deployment(
                 "SERVICE": None,
             }
         )
+        if detach:
+            click.echo("The service is running...")
+            return
+        click.echo("The service is running, press CTRL + C to stop...")
+        while True:
```

Comment:
Why is this here?

Reply:
I think its to wait until the command deliberately stopped with CTRL + C

Reply:
So @angrybayblade we are now still blocking on the deploy command but no longer stream the logs? I think there can be a benefit for this but it can also be annoying. How about we control this behaviour with a flag? By default we exit cleanly and the user has to manually stop with the stop command. When the flag is set the command blogs and `CTRL + C` ensures exit.

Reply:
Maybe the default should be the current implementation. I let you decide.